<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1455</width>
    <height>915</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(95, 193, 208, 255), stop:0.482955 rgba(196, 206, 208, 255), stop:0.982955 rgba(95, 193, 208, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>381</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Дополнительные баллы к ЕГЭ &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;в ведущих институтах страны&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="essay1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Сочинение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gto1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>ГТО</string>
    </property>
   </widget>
   <widget class="QPushButton" name="medal1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Медаль</string>
    </property>
   </widget>
   <widget class="QPushButton" name="spo1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>СПО</string>
    </property>
   </widget>
   <widget class="QPushButton" name="portfolio1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Портфолио</string>
    </property>
   </widget>
   <widget class="QPushButton" name="volunteering1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>610</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Волонтерство</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>881</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>  *при поступлении максимально
 засчитывается только 10 баллов </string>
    </property>
   </widget>
   <widget class="QLabel" name="text1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>1201</width>
      <height>521</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MainPage">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>620</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>вернуться 
на главную</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>70</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-style:italic;&quot;&gt;Отсортировать по:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>470</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите город</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sorting">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>540</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Отсортировать</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalWidget" native="true">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>110</y>
      <width>291</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="essay2">
       <property name="text">
        <string>Баллы за сочинение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="gto2">
       <property name="text">
        <string>Баллы за ГТО</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="medal2">
       <property name="text">
        <string>Баллы за медаль</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="spo2">
       <property name="text">
        <string>Баллы за СПО</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="portfolio2">
       <property name="text">
        <string>Баллы за портфолио</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="volunteering2">
       <property name="text">
        <string>Баллы за волонтёрство</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="picture">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>100</y>
      <width>331</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>710</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="add_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>710</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить университет можно сочетанием клавишь Shift+&quot;A&quot; или нажав на кнопку -&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>810</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QLabel" name="delete_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>760</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить университет можно сочетанием клавишь Shift+&quot;D&quot; или нажав на кнопку -&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="update_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>810</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить данные в университете можно сочетанием клавишь Shift+&quot;U&quot; или нажав на кнопку -&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>760</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1455</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
